src/gui/windows/save_selection.zig

```diff
@@ -80,8 +80,10 @@ fn openFolder(namePtr: usize) void {
 	const nullTerminatedName: [*:0]const u8 = @ptrFromInt(namePtr);
 	const name = std.mem.span(nullTerminatedName);
 
-	const command = if(builtin.os.tag == .windows) .{"start", "explorer"} else .{"open"};
-	const path = std.fmt.allocPrint(main.stackAllocator.allocator, "saves/{s}", .{name}) catch unreachable;
+	const command = if(builtin.os.tag == .windows) .{"explorer"} else .{"open"};
```

Comment:
Wait, but this blocks the gui until the explorer is closed. That's why I used `start` in the first place.

Reply:
weird, i can still play cubyz normally even with the explorer open

Reply:
Ok, well on wine it completely blocks the UI. But if it works fine on normal windows then I guess it isn't an issue.